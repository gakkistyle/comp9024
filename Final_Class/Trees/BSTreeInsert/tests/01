6 3 1 4 8 10
7
