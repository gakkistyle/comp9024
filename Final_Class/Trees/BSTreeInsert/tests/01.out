Enter the preorder traversal of the BST: Enter value to insert: BST before inserting 7:

     6
    / \
   /   \
  3     8
 / \     \
1   4    10

BST after inserting 7:

      6
     / \
    /   \
   /     \
  3       8
 / \     / \
1   4   7  10

